python/darpinstances/instance.py

```diff
@@ -179,6 +178,51 @@ def load_vehicles(vehicles_path: str) -> List[Vehicle]:
     return vehicles
 
 
+equipment_list = [
+    "NONE",
+    "STANDARD_SEAT",
+    "WHEELCHAIR",
+    "ELECTRIC_WHEELCHAIR",
+    "MEDICAL_STROLLER",
+]
+
+
+def get_equipment_int(str):
```

Comment:
Why is this function named get_equipment_int if it returns a string?

Reply:
it does not return a string, it returns an int